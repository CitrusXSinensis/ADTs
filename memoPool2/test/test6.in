ra100x
ca
aa 5
ca
pa
aa 12
aa 14
ca
pa
fa 8
fa 14
fa 100
pa
ca
