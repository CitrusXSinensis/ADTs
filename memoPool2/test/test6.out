1/4
2/4
100 5
4/4
100 5 12 14
Chunk: 100
Chunk: 14
Nullptr
5 12
2/4
